frontend/tests/e2e/tests/mspcontracts/GetGroupMembersContractAddressTest.js

```diff
@@ -125,7 +117,7 @@ test('Check properly filled form passes validation and no results shown when mor
     .eql(0)
 })
 
-test('Check properly filled form passes validation and displays no result if coverage not exists', async (t) => {
+test.skip('Check properly filled form passes validation and displays no result if coverage not exists', async (t) => {
```

Comment:
I think I'd rather see these tests fails with a comment as to why they are failing (need new test data). I find that skipped tests never get updated.